library/spark

```diff
@@ -1,44 +1,104 @@
 Maintainers: Apache Spark Developers <dev@spark.apache.org> (@ApacheSpark)
 GitRepo: https://github.com/apache/spark-docker.git
 
-Tags: 3.5.1-scala2.12-java17-python3-ubuntu, 3.5.1-java17-python3, 3.5.1-java17, python3-java17
+Tags: 4.0.0-preview1-scala2.13-java17-python3-ubuntu, 4.0.0-preview1-python3, 4.0.0-preview1
 Architectures: amd64, arm64v8
-GitCommit: 4f2d96a415c89cfe0fde89a55e9034d095224c94
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./4.0.0-preview1/scala2.13-java17-python3-ubuntu
+
+Tags: 4.0.0-preview1-scala2.13-java17-r-ubuntu, 4.0.0-preview1-r
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./4.0.0-preview1/scala2.13-java17-r-ubuntu
+
+Tags: 4.0.0-preview1-scala2.13-java17-ubuntu, 4.0.0-preview1-scala
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./4.0.0-preview1/scala2.13-java17-ubuntu
+
+Tags: 4.0.0-preview1-scala2.13-java17-python3-r-ubuntu
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./4.0.0-preview1/scala2.13-java17-python3-r-ubuntu
+
+Tags: 3.5.2-scala2.12-java17-python3-ubuntu, 3.5.2-java17-python3, 3.5.2-java17, python3-java17
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java17-python3-ubuntu
+
+Tags: 3.5.2-scala2.12-java17-r-ubuntu, 3.5.2-java17-r
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java17-r-ubuntu
+
+Tags: 3.5.2-scala2.12-java17-ubuntu, 3.5.2-java17-scala
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java17-ubuntu
+
+Tags: 3.5.2-scala2.12-java17-python3-r-ubuntu
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java17-python3-r-ubuntu
+
+Tags: 3.5.2-scala2.12-java11-python3-ubuntu, 3.5.2-python3, 3.5.2, python3, latest
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java11-python3-ubuntu
+
+Tags: 3.5.2-scala2.12-java11-r-ubuntu, 3.5.2-r, r
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java11-r-ubuntu
+
+Tags: 3.5.2-scala2.12-java11-ubuntu, 3.5.2-scala, scala
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java11-ubuntu
+
+Tags: 3.5.2-scala2.12-java11-python3-r-ubuntu
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
+Directory: ./3.5.2/scala2.12-java11-python3-r-ubuntu
+
+Tags: 3.5.1-scala2.12-java17-python3-ubuntu, 3.5.1-java17-python3, 3.5.1-java17
+Architectures: amd64, arm64v8
+GitCommit: b9f1f8e8ebed1959c2be3864a114b52f67519092
 Directory: ./3.5.1/scala2.12-java17-python3-ubuntu
 
 Tags: 3.5.1-scala2.12-java17-r-ubuntu, 3.5.1-java17-r
```

Comment:
https://github.com/docker-library/official-images/pull/16325#pullrequestreview-1909605636 :eyes:

(3.5.1 and 3.5.0 should not still be listed with 3.5.2 released)

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Thanks @tianon, I removed 3.5.1 and 3.5.0